bodhi/client/__init__.py

```diff
@@ -1009,11 +1011,12 @@ def releases():
 @releases.command(name='create')
 @handle_errors
 @add_options(release_options)
-def create_release(user, password, url, debug, **kwargs):
+def create_release(user, password, url, debug, not_composed_by_bodhi, **kwargs):
     """Create a release."""
     client = bindings.BodhiClient(base_url=url, username=user, password=password,
                                   staging=kwargs['staging'])
     kwargs['csrf_token'] = client.csrf()
+    kwargs['composed_by_bodhi'] = not not_composed_by_bodhi
```

Comment:
If you like my suggestion above, you can change this line to be a little more readable:

```suggestion
    kwargs['composed_by_bodhi'] = composed_by_bodhi
```